roles/openshift_facts/tasks/main.yml

```diff
@@ -6,6 +6,25 @@
     - ansible_version | version_compare('1.9.0', 'ne')
     - ansible_version | version_compare('1.9.0.1', 'ne')
 
+- name: Determine if Atomic
+  stat: path=/run/ostree-booted
+  register: s
+  changed_when: false
+
+- name: Init the is_atomic and is_containerized fact
+  set_fact:
+    is_atomic: false
+    is_containerized: false
+
+- name: Set the is_atomic fact
+  set_fact:
+    is_atomic: true
+  when: s.stat.exists
+
+- name: Set the is_containerized fact
+  set_fact:
+    is_containerized: "{{ is_atomic or containerized | default(false) | bool }}"
+    
```

Comment:
This may cause an issue with the etcd role, but I've been thinking that we need to extend the etcd role with openshift_etcd where we can safely set the openshift_* role dependencies and configuring anything openshift specific there.